article.Rmd

```diff
@@ -65,9 +65,9 @@ Evidence is needed to ensure that such investment is spent effectively and where
 Most pop-up active transport infrastructure can be classified into three broad categories:
 
 
-1. 'filtered permeability', where measures such as point closures or contraflow cycle lanes are used to design street networks in which routes for cyclists are faster and more direct than routes for drivers, e.g. as [shown](https://twitter.com/TowerHamletsNow/status/1257564043856019458) in [Tower Hamlets]
-2. banning cars and to pedestrianise streets, as in New York's 'Open Streets' scheme [@litman_pandemicresilient_2020]
-3. the reallocation of one or more lanes on wide roads to create pop-up cycleways and pavements [@orsman_covid_2020].
+1. Measures such as point closures or contraflow cycle lanes, which can be used to promote 'filtered permeability', a strategy in which street networks are redesigned so that routes for cyclists are faster and more direct than routes for drivers. An example of this is  [shown](https://twitter.com/TowerHamletsNow/status/1257564043856019458) in [Tower Hamlets].
+2. Measures to close roads entirely to particular classes of vehicle, including time-limited closures, as in New York's 'Open Streets' scheme [@litman_pandemicresilient_2020].
```

Comment:
I suggest something like:

```
 2. Measures to close roads entirely to motor traffic, permanently or during certain hours, as in New York's 'Open Streets' scheme [@litman_pandemicresilient_2020]. 
```

"motor traffic" is clearer than "particular classes of vehicle" and a little more concise.

Reply:
Good point.

Reply:
Even more concise: "cars".